src/azure-cli/azure/cli/command_modules/cosmosdb/commands.py

```diff
@@ -62,6 +83,21 @@ def load_command_table(self, _):
         g.custom_command('update', 'cli_cosmosdb_update')
         g.custom_command('list', 'cli_cosmosdb_list')
 
+    with self.command_group('cosmosdb private-endpoint-connection',
+                            cosmosdb_private_endpoint_connections_sdk,
+                            client_factory=cf_db_private_endpoint_connections) as g:
+        g.custom_command('approve', 'approve_private_endpoint_connection',
+                         validator=validate_private_endpoint_connection_id)
+        g.custom_command('reject', 'reject_private_endpoint_connection',
+                         validator=validate_private_endpoint_connection_id)
+        g.command('delete', 'delete', validator=validate_private_endpoint_connection_id)
+        g.show_command('show', 'get', validator=validate_private_endpoint_connection_id)
+
+    with self.command_group('cosmosdb private-link-resource',
+                            cosmosdb_private_link_resources_sdk,
+                            client_factory=cf_db_private_link_resources) as g:
+        g.show_command('show', 'list_by_database_account')
```

Comment:
Should we call it `list`? and please add transformer to it if the output with `value` as key?

Reply:
Added transformer.